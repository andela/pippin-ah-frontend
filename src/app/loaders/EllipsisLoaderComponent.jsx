import React from 'react';
import { ellipsisLoader } from '../../img';

const EllipsisLoaderComponent = () => {
  return <img className="ellipsis" src={ellipsisLoader} alt="loader" />;
};

export default EllipsisLoaderComponent;
