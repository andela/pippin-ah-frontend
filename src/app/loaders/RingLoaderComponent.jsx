import React from 'react'
import '../../styles/loader.css'

const RingLoaderComponent = () => {
    return (
        <div className="lds-ring">
            <div />
            <div />
            <div />
            <div />
            <div />
        </div>
    )
}

export default RingLoaderComponent
