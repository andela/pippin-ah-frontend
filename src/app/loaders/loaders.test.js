import React from 'react';
import { shallow } from 'enzyme';
import RingLoaderComponent from './RingLoaderComponent';
import EllipsisLoaderComponent from './EllipsisLoaderComponent';

describe('It should render the RingLoader', () => {
  it('should render the RingLoaderComponent', () => {
    const component = shallow(<RingLoaderComponent />);
    expect(component.exists()).toBe(true);
    expect(component).toMatchSnapshot();
  });
});

describe('It should render the EllipsisLoader', () => {
  it('should render the EllipsisLoaderComponent', () => {
    const component = shallow(<EllipsisLoaderComponent />);
    expect(component.exists()).toBe(true);
    expect(component).toMatchSnapshot();
  });
});
