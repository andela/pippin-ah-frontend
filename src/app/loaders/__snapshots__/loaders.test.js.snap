// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It should render the RingLoader should render the Landing Page 1`] = `
<div
  className="lds-ring"
>
  <div />
  <div />
  <div />
  <div />
  <div />
</div>
`;
